nautobot/utilities/testing/views.py

```diff
@@ -141,7 +141,12 @@ def test_get_object_without_permission(self):
 
             # Try GET without permission
             with disable_warnings("django.request"):
-                self.assertHttpStatus(self.client.get(instance.get_absolute_url()), [403, 404])
+                response = self.client.get(instance.get_absolute_url())
+                self.assertHttpStatus(response, [403, 404])
+                response_body = response.content.decode(response.charset)
+                self.assertNotIn(
+                    "/login/?next=" + self._get_queryset().first().get_absolute_url(), response_body, msg=response_body
```

Comment:
Why `self._get_queryset().first().get_absolute_url()` rather than `instance.get_absolute_url()`?

Or for that matter, why not just `self.assertNotIn("/login/", response_body, msg=response_body)`?